RecoTracker/IterativeTracking/python/dnnQualityCuts.py

```diff
@@ -0,0 +1,13 @@
+qualityCutDictionary = {
```

Comment:
I would ask to put a reference/indico URL to the slides where these numbers were presented.

Reply:
Please add a reference URL here.